### PR TITLE
TRIVIAL: update MAINTENANCE.md

### PR DESCRIPTION
**Pull request by Mara3l**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: MAINTENANCE.md [hunk 1]
```diff
@@ -3,11 +3,20 @@
...
+* Once it is merged:
+  * Checkout latest master and tag it vX.Y.Z
 * Push the tag to the gooddata/gooddata-python-sdk repository (e.g. `git push <remote> vX.Y.Z`)
 
+### Releasing documentation
+
+* Create new branch named `rel/X.Y.Z`
+  * This branch now hosts the content for documentation of the X.Y.Z version. Any changes to the content in the branch will be public.
+  * All the styling of the documentation is taken from the `master` branch. For more details see `./scripts/generate.sh`.
+* Make the `rel/X.Y.Z` branch protected.
```
Comment (hkad98) [inline MAINTENANCE.md:17]: This is not necessary. There is a regex for rel/.* that makes these branches protected by default.

File: MAINTENANCE.md [hunk 2]
```diff
@@ -3,11 +3,20 @@
...
+* Dispatch the Github workflow `netlify-deploy`
```
Comment (hkad98) [inline MAINTENANCE.md:18]: and select rel/x.y.z version?

File: MAINTENANCE.md [hunk 3]
```diff
@@ -3,11 +3,20 @@
...
-* Create pull request with the latest commit with bumped versions
-* Ask for merge of pull request. Once it is merged:
-* Checkout latest master tag it vX.Y.Z
+* Create a pull request with the latest commit with bumped versions
+* Ask for merge of a pull request.
+* Once it is merged:
+  * Checkout latest master and tag it vX.Y.Z
 * Push the tag to the gooddata/gooddata-python-sdk repository (e.g. `git push <remote> vX.Y.Z`)
 
+### Releasing documentation
```
Comment (hkad98) [inline MAINTENANCE.md:12]: I think it would make sense to do this before executing `make release` WDYT?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: MAINTENANCE.md (hunk 2 above)
Comment (Mara3l) [reply MAINTENANCE.md:17]: Actually no.. this releases all the versions.

The branch to be selected has to be master, as the selected branch determines the styling, while the `rel/x.y.z` branches determine the content.

Comment (hkad98) [reply MAINTENANCE.md:17]: Ahh okay. Please add this information there.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by hkad98 — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR229

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul/62/ : SUCCESS in 1m 15s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/212/ : SUCCESS in 58s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/215/ : SUCCESS in 13s

**Pull request merged**